frontend/app/ui_components/wiki-toolbar-directive.js

```diff
@@ -27,7 +27,7 @@
 //++
 
 module.exports = function() {
-  var HELP_LINK_HTML = '<a href="/help/wiki_syntax" title="' + I18n.t('js.inplace.link_formatting_help') + '" class="icon icon-help" onclick="window.open(&quot;/help/wiki_syntax&quot;, &quot;&quot;, &quot;resizable=yes, location=no, width=600, height=640, menubar=no, status=no, scrollbars=yes&quot;); return false;"><span class="hidden-for-sighted">' + I18n.t('js.inplace.link_formatting_help') + '</span></a>',
```

Comment:
I assumed the title to suffice, but apparently I was wrong http://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/

Thanks for spotting.